src/promptflow/tests/sdk_cli_test/e2etests/test_flow_run.py

```diff
@@ -1265,3 +1265,21 @@ def test_eager_flow_test_invalid_cases(self, pf):
                 data=f"{DATAS_DIR}/simple_eager_flow_data.jsonl",
             )
         assert "'path': ['Missing data for required field.']" in str(e.value)
+
```

Comment:
Can we add an "pf run list" e2e test?

Reply:
I think current test already covered the "pf run list"'s scenario. It's hard to create a incomplete run and make sure list can fetch it since we only list top 50 by default.

Reply:
Can we create a run before the pf run list? Then delete the flow path? Then we can test the problem we encountered this time?

Reply:
Does list result guarantee to return in creation order?

Reply:
The newly added CLI test will cover other code paths in list scenario.